src/components/ui/UiTrambusLogo.vue

```diff
@@ -0,0 +1,21 @@
+<script lang="ts" setup>
+import trambusLogo from '@/assets/icons/trambusLogo.png'
+</script>
+
+<template>
+  <div class="flex flex-col items-start p-0 gap-2 w-4/5 h-28">
+    <div class="flex flex-row items-start p-0 gap-2 h-14">
+      <div class="h-14 w-14 flex flex-col justify-center">
+        <img :src="trambusLogo" />
+      </div>
+      <div
+        class="font-poppins font-bold text-4xl w-80 h-14 justify-center flex flex-col"
+      >
+        Reseau Trambus
+      </div>
+    </div>
+    <div class="font-poppins font-light text-base leading-6">
```

Comment:
Try to apply the css at higher level, remember it's **cascading**
`font-poppins` should not be repeated all the way down.